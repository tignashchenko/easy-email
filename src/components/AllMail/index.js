import React from 'react';

const AllMail = () => (
    <div>
        <h1>This is All Mail!</h1>
    </div>
);

export default AllMail;
