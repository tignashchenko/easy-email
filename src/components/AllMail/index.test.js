import React from 'react';
import dom from 'react-test-renderer';
import AllMail from './';

const renderTree = dom.create(<AllMail />).toJSON();

test('AllMail component should correspond to its snapshot counterpart', () => {
    expect(renderTree).toMatchSnapshot();
});
