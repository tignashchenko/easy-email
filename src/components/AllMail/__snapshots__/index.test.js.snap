// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllMail component should correspond to its snapshot counterpart 1`] = `
<div>
  <h1>
    This is All Mail!
  </h1>
</div>
`;
